variables.tf

```diff
@@ -436,6 +436,12 @@ variable "default_node_pool_fips_enabled" {
   description = " (Optional) Should the nodes in this Node Pool have Federal Information Processing Standard enabled? Changing this forces a new resource to be created."
 }
 
+variable "default_node_pool_gpu_instance" {
```

Comment:
The default nodepool is a system node pool that contains also AKS system workloads. I don't see the value of adding a GPU workload on this nodepool.

I suggest to enable the GPU features only on the extra nodepools.

Why your use case needs a GPU on the system node pool ? Could you please share more about your scenario ? Thanks